packages/litellm/custom/langflow_handler.py

```diff
@@ -171,6 +173,18 @@ class Langflow(CustomLLM):
     def __init__(self):
         self.mapping_endpoint = f'{os.environ["HELPER_BACKEND"]}/mapping'
 
+    def _get_langflow_url(self, api_base: str) -> str:
+        """Convert external URLs to internal Docker container URLs for proper networking."""
+        if 'localhost' in api_base or '127.0.0.1' in api_base:
```

Comment:
This seems a bit arbitary. It relies on 
1. Assuming this is always run within Docker
2. The name of the service will always be `langflow`